src/main/java/com/minecolonies/colony/permissions/Permissions.java

```diff
@@ -180,12 +183,12 @@ public static Rank getDemotionRank(Rank rank)
     public void loadPermissions(@NotNull NBTTagCompound compound)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "loadPermissions" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)